### ISSUE TITLE
Update Architecture documents (once code is merged into main branch)

### ISSUE THREAD

**Issue by beno11224** [Brookke/Lorem-Ipsum#37]
2. Architecture report [22 marks]:
- a) Give a concrete architecture showing the structure of the team’s code, with a brief statement of the specific languages used to describe the architecture, and, if appropriate, the tool(s) used to create the concrete architecture representation. (12 marks, ≤ 2 page)
- b) Give a systematic justification for the concrete architecture, explaining how the concrete architecture builds from the abstract architecture – and any changes that had to be made to the abstract architecture. Relate the concrete architecture clearly to the requirements, using your requirements referencing for identification, and consistent naming of constructs to provide traceability. Provide precise URLs to any relevant web pages. (10 marks, ≤ 2 pages)
